.clang-format

```diff
@@ -9,11 +9,11 @@ AlignEscapedNewlines: Right
 AlignOperands:   true
 AlignTrailingComments: true
 AllowAllParametersOfDeclarationOnNextLine: true
-AllowShortBlocksOnASingleLine: false
+AllowShortBlocksOnASingleLine: Never
 AllowShortCaseLabelsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: Inline
 AllowShortIfStatementsOnASingleLine: true
-AllowShortLoopsOnASingleLine: true
```

Comment:
**Review note:** This was changed, as the behavior of `AllowShortLoopsOnASingleLine` was changed since this was `true`. In order for formatting to remain the same as it currently is, `AllowShortLoopsOnASingleLine` should be `false`.